packages/cubejs-client-vue/src/QueryBuilder.js

```diff
@@ -392,11 +392,17 @@ export default {
     this.copyQueryFromProps();
 
     if (isQueryPresent(this.initialQuery)) {
-      const dryRunResponse = await this.cubejsApi.dryRun(this.initialQuery);
-      this.pivotConfig = ResultSet.getNormalizedPivotConfig(
-        dryRunResponse?.pivotQuery || {},
-        this.pivotConfig
-      );
+      await this.cubejsApi
+        .dryRun(this.initialQuery)
+        .then((dryRunResponse) => {
```

Comment:
Let's refactor it a bit to use `try/catch` syntax here to make it clearer

Reply:
done!